### DISCUSSION TITLE
Plex as music provider

### DISCUSSION THREAD

**Discussion by bckthomas** [music-assistant/support#482] [Feature requests and ideas]
Hi
I would like to use plex as music provider, Is it possible to do that ?
Thanks

## PLEX IS AVAILABLE https://music-assistant.io/music-providers/plex/

**Comment by Daniel15** [comment]
This would be great. It fits in with the Home Assistant philosophy of self-hosting rather than relying on third parties :)

I use Plexamp on my phone and PC so it'd be useful to support Plex in Music Assistant too.

**Comment by RJ-Make** [comment]
I also like this idea.

**Comment by RickeyWard** [comment]
This would be great!

**Comment by smarthomejunkie** [comment]
+1 from me :)

**Comment by snickers2k** [comment]
+1

**Comment by fribse** [comment]
I had the same thought when I installed Music Assistant, great supplement.

**Comment by RemakingEden** [comment]
Definitely a +1 from me

**Comment by gzawahra** [comment]
defenitly a great idea, i thought it already had it since plex is integrated into hass...

**Reply by gravyflex** [reply]
-All of my music is in Plex. This would be a great addition-

Scratch that, Enabled DLNA on the Plex Server and this achieves the goal.

**Comment by Jamsgrove** [comment]
I am also keen as for this feature

**Comment by Scirazza** [comment]
And if its possible to control it with plexamp and send it to piCoreplayer or release some multiroom options. THAT would be great!

**Comment by OzGav** [comment]
Plex Connect support is available on 2.7.0 beta 16 and later

**Comment by fakethinkpad85** [comment]
Any chance we can have the ability to add tracks to Plex playlists from MA? right now we only get "NotImplemented" message.

**Reply by OzGav** [reply]
@anatosun

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
